admin/views/user-profile.php

```diff
@@ -5,86 +5,49 @@
 
 ?>
 
-<h2 id="wordpress-seo"><?php
-	/* translators: %1$s expands to Yoast SEO */
-	printf( __( '%1$s settings', 'wordpress-seo' ), 'Yoast SEO' );
-	?></h2>
-<table class="form-table">
-	<tr>
-		<th scope="row">
-			<label
-				for="wpseo_author_title"><?php _e( 'Title to use for Author page', 'wordpress-seo' ); ?></label>
-		</th>
-		<td>
-			<input class="regular-text" type="text" id="wpseo_author_title" name="wpseo_author_title"
-		           value="<?php echo esc_attr( get_the_author_meta( 'wpseo_title', $user->ID ) ); ?>"/>
-		</td>
-	</tr>
-	<tr>
-		<th scope="row">
-			<label
-				for="wpseo_author_metadesc"><?php _e( 'Meta description to use for Author page', 'wordpress-seo' ); ?></label>
-		</th>
-		<td>
-			<textarea rows="3" cols="30" id="wpseo_author_metadesc"
-			          name="wpseo_author_metadesc"><?php echo esc_textarea( get_the_author_meta( 'wpseo_metadesc', $user->ID ) ); ?></textarea>
-		</td>
-	</tr>
+<div class="yoast yoast-settings">
+
+	<h2 id="wordpress-seo"><?php
+		/* translators: %1$s expands to Yoast SEO */
+		printf( __( '%1$s settings', 'wordpress-seo' ), 'Yoast SEO' );
+		?></h2>
```

Comment:
Why has the title been moved into the form when it used to be outside? It looks fine, but it does break pattern. Does it improve something i'm not thinking of?